py3status/modules/gitlab.py

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8 -*-
+"""
+Display number of issues, requests and more from a Gitlab project.
+
+A token is required. See https://gitlab.com/profile/personal_access_tokens
+to make one. Pick a name, eg py3status and enable api in scopes. Save.
+
+Configuration parameters:
+    auth_token: specify a personal access token (default None)
+    button_action: mouse button to take action (default 1)
+    button_refresh: mouse button to refresh this module (default 2)
+    cache_timeout: refresh interval for this module (default 60)
+    format: display format for this module
+        (default '{name} {open_issues_count}/{open_merge_requests_count}')
+    url: specify a project url to track (default None)
```

Comment:
Is this enough?  With the github module we need this split so that we can do the any notifications vs just project ones.

Does it give us enough flexibility for self hosted setups?